client/pubspec.yaml

```diff
@@ -53,6 +53,7 @@ dependencies:
   # TODO: upgrade to ^0.6.5+2, build warning treated as error
   share: 0.6.3+6
   shared_preferences: ^0.5.6
+  timeago: ^2.0.28
```

Comment:
Much better to use a library....

Reply:
I've used this before, the code is relatively simple but it works